package.json

```diff
@@ -79,7 +79,7 @@
     "core-js": "^3.29.1",
     "cross-env": "7.0.3",
     "css-loader": "6.8.1",
-    "cypress": "13.6.0",
+    "cypress": "^13.6.3",
```

Comment:
I suspect [renovate will revert that change](https://github.com/renovatebot/renovate/blob/main/docs/usage/dependency-pinning.md) in the next renovate PR.

Reply:
Alright. I'll update it with a pinned version :smile: